package.json

```diff
@@ -45,7 +45,7 @@
     "clean-css-cli": "^4.3.0",
     "jest": "^27.5.1",
     "node-plop": "^0.26.2",
-    "node-sass": "^4.14.1",
+    "sass": "1.49.9",
```

Comment:
Let's use the latest version (`1.51.0` as of this message).

Reply:
done